sdk/protocol2.go

```diff
@@ -132,6 +132,19 @@ func (d *EntityData) NewMetricSet(eventType string) metric.MetricSet {
 	return ms
 }
 
+// Event method creates or retrieves an already created Event.
+func (d *EntityData) Event(summary, category string) *Event {
+	for _, e := range d.Events {
```

Comment:
I think we shouldn't check if an event is already present. Imagine you're generating an event to know if a service has been restarted: If during the interval you're monitoring events, the service has been restarted 3 times, probably you would like to have those 3 events.

Reply:
Yes, you're right!